eks_infrastructure/patch_hosts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
was this file pushed with +x chmod permissions?

Reply:
yes, this file has executable permission.